script.js

```diff
@@ -0,0 +1,94 @@
+  const speakerInfo2 = new Speaker('Hon. Ursula Owusu', 'Minister of Communication and Digital Economy', 'A Design Strategist and a Tech Enthusiast', 'img/ursula-owusu.hpeg.jpg');
+  const speakerInfo3 = new Speaker('Hon. Dr. Kwaku Afriyie', 'Minister, Environment, Science, Technology and Innovation (MESTI)', 'A Design Strategist and a Tech Enthusiast', 'img/hon-kwaku-afriyie.jpeg');
+  const speakerInfo4 = new Speaker('Dr Gordon Adomdza', 'Associate Professor of Innovation and Entrepreneurship', 'A Design Strategist and a Tech Enthusiast', 'img/gordon.jpg');
+  speakers = [speakerInfo1, speakerInfo2, speakerInfo3, speakerInfo4];
+  return speakers;
+}
+
+function prepareSpeaker(speakers) {
+  const speakerSection = document.getElementById('speakers');
+  const buttomDiv = document.createElement('div');
+  const bottomP = document.createElement('p');
+  bottomP.innerHTML = 'MORE';
+  const bottomI = document.createElement('i');
+  bottomI.classList.toggle('fa-solid', 'fa-chevron-down');
+  bottomI.classList.toggle('fa-chevron-down');
+  buttomDiv.classList.toggle('border-speakers');
+  const heading = document.createElement('h3');
+  const divider = document.createElement('hr');
+  speakerSection.appendChild(heading);
+  speakerSection.appendChild(divider);
+  divider.classList.toggle('divider');
+  const speakersContainer = document.createElement('div');
+  speakersContainer.classList.toggle('speakers');
+  heading.innerHTML = 'Featured Speakers';
+  for (let i = 0; i < speakers.length; i += 1) {
+    const speakerContainer = document.createElement('div');
+    speakerContainer.classList.toggle('speaker');
+    const speakerImage = document.createElement('img');
+    const speakerDetailsContainer = document.createElement('div');
+    const speakerName = document.createElement('h3');
+    const speakerTitle = document.createElement('cite');
+    speakerTitle.classList.toggle('subtitle');
+    const speakerTitleDivider = document.createElement('div');
+    speakerTitleDivider.classList.toggle('divide-cite');
+    const speakerDescription = document.createElement('p');
+
+    // Set Values to the elements
+    speakerImage.src = speakers[i].img;
+    speakerName.innerHTML = speakers[i].fullname;
+    speakerTitle.innerHTML = speakers[i].title;
+    speakerDescription.innerHTML = speakers[i].desc;
+
+    // Append Elements to their right Div
+    speakerContainer.appendChild(speakerImage);
+    speakerDetailsContainer.appendChild(speakerName);
+    speakerDetailsContainer.appendChild(speakerTitle);
+    speakerDetailsContainer.appendChild(speakerTitleDivider);
+    speakerDetailsContainer.appendChild(speakerDescription);
+    speakerContainer.appendChild(speakerDetailsContainer);
+
+    // Attach to main Section
+    speakersContainer.appendChild(speakerContainer);
+    speakerSection.append(speakersContainer);
+  }
+  buttomDiv.appendChild(bottomP);
+  buttomDiv.appendChild(bottomI);
+  speakerSection.append(buttomDiv);
```

Comment:
- [OPTIONAL] I think is a good idea to refactor this block of code using the innerHTML property. This way you can insert directly HTML code avoiding the creation of these elements step by step using appendChild. You can read more about [this here.](https://www.w3schools.com/jsref/prop_html_innerhtml.asp) :nerd_face: :nerd_face:

Reply:
I read [somewhere](https://medium.com/@jenlindner22/the-risk-of-innerhtml-3981253fe217) that innerHtml is not secured, perhaps you can share your opinion on this as well.

Reply:
> I read [somewhere](https://medium.com/@jenlindner22/the-risk-of-innerhtml-3981253fe217) that innerHtml is not secured, perhaps you can share your opinion on this as well.

I understand your point, but you can go through the link you read somewhere again. It will help you to understand that there are certain circumstances that you can choose to use the innerHTML. Though it still remains a personal decision.
Kindly check an excerpts of the link you shared below for more info. :+1: 

_Although the use of Node.innerHTML is not completely safe in all use cases, `it can still be used in some circumstances`, `such as inserting static data on a page where user input is not collected!` For the purposes of our projects at the Flatiron School, an XSS attack would not be much more than a small annoyance, as `we aren’t dealing with personal or sensitive data at this stage`._

_However, for a website handling personal information, an XSS attack can be significantly detrimental to those maintaining the data and its users. For those creating web pages which will be handling personal information or undergoing a security review, Node.textContent should be used in lieu of Node.innerHTML as an extra precaution against an attack._